bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "stattracker-gui",
+  "description": "Project with Python",
+  "main": "index.html",
+  "authors": [
+    "Shay Hall"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/tiyd-stattracker/stattracker-gui",
+  "moduleType": [],
+  "private": true,
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "dependencies": {
+    "normalize-css": "~3.0.3",
+    "modernizr": "~3.1.0",
```

Comment:
Check out https://github.com/components/modernizr for a wiredep-friendly version of Modernizr